docs/understanding-airbyte/supported-data-types.md

```diff
@@ -17,14 +17,16 @@ This table summarizes the available types. See the [Specific Types](#specific-ty
 | Datetime with timezone                                         | `{"type": "string", "format": "date-time", "airbyte_type": "timestamp_with_timezone"}`    | `"2022-11-22T01:23:45+05:00"`                                                   |
 | Datetime without timezone                                      | `{"type": "string", "format": "date-time", "airbyte_type": "timestamp_without_timezone"}` | `"2022-11-22T01:23:45"`                                                         |
 | Integer                                                        | `{"type": "integer"}`                                                                    | `42`                                                                            |
-| Big integer (unrepresentable as a 64-bit two's complement int) | `{"type": "string", "airbyte_type": "big_integer"}`                                      | `"123141241234124123141241234124123141241234124123141241234124123141241234124"` |
+| Big integer (unrepresentable as a 64-bit two's complement int) | `{"type": "string", "airbyte_type": "big_integer"}`                                      | `"12345678901234567890123456789012345678"` |
 | Number                                                         | `{"type": "number"}`                                                                     | `1234.56`                                                                       |
 | Big number (unrepresentable as a 64-bit IEEE 754 float)        | `{"type": "string", "airbyte_type": "big_number"}`                                       | `"1,000,000,...,000.1234"` with 500 0's                                         |
 | Array                                                          | `{"type": "array"}`; optionally `items` and `additionalItems`                            | `[1, 2, 3]`                                                                     |
 | Object                                                         | `{"type": "object"}`; optionally `properties` and `additionalProperties`                 | `{"foo": "bar"}`                                                                |
 | Untyped (i.e. any value is valid)                              | `{}`                                                                                     |                                                                                 |
 | Union                                                          | `{"anyOf": [...]}` or `{"oneOf": [...]}`                                                 |                                                                                 |
 
+Note that some of these may be destination-dependent. For example, Snowflake `NUMERIC` columns can be at most 38 digits wide, but Postgres `NUMERIC` columns may have up to 131072 digits before the decimal point.
```

Comment:
this is something that needs to be addressed as part of the types project. Kind of similar to https://github.com/airbytehq/airbyte/issues/14756, expect that numbers are typically better at having easily-defined ranges, so we can probably handle these more smartly.